menu/menu_defines.h

```diff
@@ -429,6 +429,7 @@ enum ozone_color_theme
    OZONE_COLOR_THEME_HACKING_THE_KERNEL,
    OZONE_COLOR_THEME_TWILIGHT_ZONE,
    OZONE_COLOR_THEME_DRACULA,
+   OZONE_COLOR_THEME_SELENIUM,
```

Comment:
FYI adding this in the middle of the list breaks existing configs that use the themes after it..

Reply:
Please fix it before the next stable.

Reply:
An apology, I thought that with the review you had corrected it, in any case, the pull request has already been made and the only way to edit it seems to be by opening another pull request, do I do that?

Reply:
Yes, moving it to the end before `LAST` will do the job for the time being.

Would be great if in the future we would be able to get rid of all of these integer based config values and use strings instead, so the order does not matter, and thus they can finally be in alphabetic order.